.github/workflows/ci.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
   push:
     branches: [ main ]
+  workflow_call:
```

Comment:
is this really necessary?

Reply:
yes it is failing when removed: https://github.com/codeforamerica/pya/actions/runs/15500870230

added it back in!

Reply:
i was pretty sure, but since this has been a lot guess and check, i just wanted to double check.

Reply:
Thanks for checking!!!

https://docs.github.com/en/actions/sharing-automations/reusing-workflows#creating-a-reusable-workflow